extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/tasks/GoogleTasksExporter.java

```diff
@@ -106,7 +107,9 @@ public GoogleTasksExporter(GoogleCredentialFactory credentialFactory) {
         result
             .getItems()
             .stream()
-            .map(t -> new TaskModel(resource.getId(), t.getTitle(), t.getNotes()))
+            .map(t -> new TaskModel(resource.getId(), t.getTitle(), t.getNotes(),
```

Comment:
Do we need to validate these values exist?

Reply:
Worked fine with empty-titled and empty-note tasks, so I think we don't.